ratis-server/src/test/java/org/apache/ratis/server/impl/RaftReconfigurationBaseTest.java

```diff
@@ -339,8 +336,8 @@ void runTestBootstrapReconf(int numNewPeer, boolean startNewPeer, CLUSTER cluste
         final PeerChanges c1 = cluster.addNewPeers(numNewPeer, startNewPeer);
         LOG.info("Start changing the configuration: {}",
                 asList(c1.allPeersInNewConf));
-        final AtomicReference<Boolean> success = new AtomicReference<>();
 
+        final AtomicReference<Boolean> success = new AtomicReference<>();
```

Comment:
```suggestion
        final AtomicReference<Boolean> success = new AtomicReference<>();

```
Let's revert the whitespace change.